src/app/api/community.service.ts

```diff
@@ -0,0 +1,98 @@
+import { Injectable } from '@angular/core';
+import { BaseApiService, Version } from './base-api.service';
+
+export interface Community {
+  name: string;
+  title: string;
+  description: string;
```

Comment:
`description`, `sidebar`, and `nsfw` are optional and should have a ? like: `description?: string`